My-Portfolio: Header-section completed

ometman opened the pull request:
The header and headline sections have been completed with the following:
A personalized logo and mobile menu were added.
A personalized primary H1 text, a simple introduction and social contacts.
This section was styled for mobile and desktop for responsiveness.
An MIT License and README.md were added.
Linters errors were checked and passed.

Debas-31 reviewed (commented):
Hi @ometman , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :negative_squared_cross_mark:  :negative_squared_cross_mark:  :negative_squared_cross_mark: 

- [ ] You have submitted a project PR without a README file. Please use this[ Microverse README template](https://github.com/microverseinc/readme-template) and customize it based on the project requirement.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

BregornOriginal commented on index.html:
- [ ] Good work creating a header tag, but remember that you can only have one `header` tag on your main page. So please remove the second one. Instead, you can add a `nav` tag where it's wrapping your logo and hamburger button 💯

BregornOriginal commented on index.html:
✅ Good work creating the `ul` tag wrapping `li` and `a` tags for each social icon 💯

BregornOriginal commented on index.html:
- [ ] Instead of adding each image to this section, you can use the main image that you have on the FIGMA template and add it with CSS as a background image. Please apply this on your CSS so you can manage this image in a better way and it will have the corresponding properties to fit on your mobile screen. use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

BregornOriginal reviewed (changes requested):
## Hi @ometman  

## Good job so far!
## There are some issues that you still need to work on to go to the next project but you are almost there!

![almost-over-work](https://user-images.githubusercontent.com/93630700/205944028-78d464dd-3da9-44d4-a952-939267b0372c.gif)

## Highlights

✅ Linters are passing
✅ Good README ***(KINDLY, FOR YOUR NEXT PROJECT ADD YOUR README FILE ON YOUR ACTUAL WORKING BRANCH INSTEAD OF THE MAIN LIKE YOU DID NOW.)***
✅ Good commit history
✅ Good Pull Request description


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ometman commented on index.html:
That's right. Thanks.

ometman commented on index.html:
Ok. Since it is not a single image, I will use <p>.

ometman commented:
I have updated the index.html file. I removed the one header to have only one header.  The background images of the header section have been removed and placed as background images  in  the css. Thanks for the review.

GedeonTS commented on index.html:
- [OPTIONAL] I suggest to wrap the logo with an anchor tag `<a>` instead of div, as it's always used as a link to get back to the main page :recycle: 

_Please take this requirement into account since it's optional only because the previous reviewer didn't notice it_

GedeonTS commented on index.html:
- [ ] Just like the previous reviewer pointed, you had to keep only one header container,congratulations on trying :clap: , However, I suggest to follow the structure below:

```
<header>
<a> Logo </a>
<nav>Humburger</nav>
</header>
```

GedeonTS commented on index.html:
- [ ] Kindly move the `<header>` above as per the comment above, since It must only contain the logo and nav :recycle:

GedeonTS reviewed (changes requested):
# STATUS: CHANGES REQUIRED :recycle: :

Hi @ometman , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media.giphy.com/media/t8YPf3ACKeouLzEIvr/giphy.gif">

## Highlights :tophat:  :+1: 
- Linters are passing :+1: 
- Good commit messages :+1: 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
__Kindly make sure to use github flow correctly in your next projest, since I have noticed that your two branches are have different commit history__ :no_entry_sign: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ometman commented on index.html:
Oh Ok . That's good.

ometman commented on index.html:
Please the background image covers only the head section., which includes the header and headline. If the header is moved it will mean the background image must go into a wrapper sort of within the body.  I was asked not to use that. Else,  the background image will have to go into the body, which will likely affect the additional pages in later design when the height is increased.  Please advice.

ometman commented on index.html:
@GedeonTS Please I have moved the header as requested and updated the styles.css accordingly. Thank you.

Olamarx commented on index.html:
[OPTIONAL] your link href should have a dot something like `./css/styles.css`. The css file is not linked to the webpage and as a result, it is not displaying any style.

### What I have here
![image](https://user-images.githubusercontent.com/86200356/206115430-fe4bc733-15b2-42ff-ad53-40350be2f296.png)

_Please take this requirement into account since it's optional only because the previous reviewer didn't notice it_

Olamarx reviewed (changes requested):
Hi @ometman, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
<img src="https://media2.giphy.com/media/hXz6GN0ZilLwhgLo49/giphy.gif"/>

## Highlights
- Linters passed ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly add a readme file to your project so that one would able to have an understanding of what your project entails and how to go around it. You can find it [here](https://github.com/microverseinc/readme-template)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ometman commented on index.html:
I can't tell. but one of the reviewers in commenting broke the code and I had to fix it.
There is no /. It is updated.

Abenezer-Tilahun commented on index.html:
Fixed.

ometman commented on index.html:
Thank you so much for the review.

christianbiring1 reviewed (commented):
Hi @ometman  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ometman committed:
Create index.html, styles and images for project

ometman committed:
Add .gitignore and MIT License

ometman committed:
Update readme for porfolio project

ometman committed:
Install linters for error checking

ometman committed:
Fix linter errors

ometman committed:
Fix linters errors in lighthouse

ometman committed:
Update linters.yml

brainconnect93 reviewed (changes requested):
## STATUS: Changes Required :recycle:

### Hi @ometman, 👋 👏 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media.tenor.com/isiLRnxz3zwAAAAM/cat-driving-serious.gif" width="500px" height="300px">

### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors. ✔️
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ometman committed:
Fix broken MIT License link and header structure

ometman committed:
Modify style for index.html changes

ometman committed:
Merge branch 'header-section' of https://github.com/ometman/My-Portfolio into header-section

ometman committed:
Modify styles for mobile and desktop

franclobo reviewed (changes requested):
Hi @ometman, 

# ♻️ Required Changes ♻️

![so-close-jim-halpert](https://user-images.githubusercontent.com/58642949/227408480-df62f4a0-1d39-49e4-b69a-12d339e9d1be.gif)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ometman committed:
Update styles after review

ometman committed:
Modify styles and fix license link

ometman committed:
Fix linter errors

AdnanOlarmmi reviewed (approved):
Hi @ometman,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Approved office man](https://user-images.githubusercontent.com/41163571/219113922-5007d6bb-ff59-4916-b6d3-e5aad3cb4051.gif)

## Highlights
- No linter errors ✔️
- Correct Github flow ✔️
- Descriptive PR ✔️
- Nice one implementing required changes from the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ometman commented:
Thank you
